src/pages/de/core-concepts/routing.md

```diff
@@ -0,0 +1,291 @@
+- Rest-Parameter haben die niedrigste Priorität
+- Bei Gleichstand wird alphabetisch sortiert
+
+Hier noch einige Beispiele basierend auf der obenstehenden Dateistruktur, welche Astro-Dateien zur Erzeugung des HTML-Codes für welche angeforderten URLs verwendet werden:
+
+- `pages/posts/create.astro` - Erzeugt `/posts/create`
+- `pages/posts/[pid].astro` - Erzeugt `/posts/1`, `/posts/abc` usw., aber nicht `/posts/create`
+- `pages/posts/[...slug].astro` - Erzeugt `/posts/1/2`, `/posts/a/b/c` usw., aber nicht `/posts/create`, `/posts/1` oder `/posts/abc`
+
+## Paginierung (Seiten&shy;nummerierung)
+
+Astro unterstützt von Haus aus das Aufteilen großer Datenmengen auf mehrere Seiten. Es werden dabei alle üblicherweise benötigten Eigenschaften generiert, einschließlich der URLs der vorherigen/nächsten Seite, der Gesamtzahl der Seiten und mehr.
+
+Um eine Seitennummerierung zu einer Route hinzuzufügen, kannst du die gleiche Syntax mit `[Klammern]` wie bei einer standardmäßigen dynamischen Route verwenden. Beispielsweise kann der Dateiname `/astronauten/[page].astro` die Routen für `/astronauten/1`, `/astronauten/2` usw. erzeugen, wobei `[page]` durch die generierte Seitennummer ersetzt wird.
+
+Das folgende Beispiel zeigt, wie du die Funktion `paginate()` verwenden kannst, um diese Seiten für ein Array von Werten zu generieren:
+
+```astro
+---
+// Beispiel: /src/pages/astronauten/[page].astro
+export async function getStaticPaths({ paginate }) {
+  const astronauten = [{
+    astronaut: 'Neil Armstrong',
+  }, {
+    astronaut: 'Buzz Aldrin',
+  }, {
+    astronaut: 'Sally Ride',
+  }, {
+    astronaut: 'John Glenn',
+  }];
+  // Erzeuge Seiten aus unserem Astronauten-Array, mit 2 pro Seite
+  return paginate(astronauten, { pageSize: 2 });
+}
+// Alle generierten Paginierungs-Daten werden an die
+// Eigenschaft "page" von Astro.props übergeben
+const { page } = Astro.props;
+---
+
+<!-- Zeige die aktuelle Seitennummer an.
+  Astro.params.page kann auch verwendet werden! -->
+<h1>Page {page.currentPage}</h1>
+<ul>
+  <!-- Liste die auf dieser Seite enthaltenen Astronauten auf -->
+  {page.data.map(({ astronaut }) => <li>{astronaut}</li>)}
+</ul>
+```
+
+Dadurch werden die folgenden Seiten mit jeweils 2 Astronauten pro Seite erzeugt:
+- `/astronauten/1` - Seite 1: Zeigt "Neil Armstrong" und "Buzz Aldrin" an
+- `/astronauten/2` - Seite 2: Zeigt "Sally Ride" und "John Glenn" an
+
+### Die `page`-Eigenschaft (Prop)
+
+Wenn du die `paginate()`-Funktion verwendest, werden die erzeugten Daten jeder Seite über eine Eigenschaft namens `page` übergeben. Der Wert dieser `page`-Eigenschaft ist ein Objekt mit vielen nützlichen Untereigenschaften. Hier einige Highlights:
+
+- **page.data** - Ein Array aus den Elementen, die auf der aktuellen Seite enthalten sind (eine Teilmenge der von dir an die `paginate()`-Funktion übergebenen Elemente)
+- **page.url.next** - Ein Link zur nächsten Seitennummer
```

Comment:
Seems more semantically correct to refer to the page object in the collection, imao.
```suggestion
- **page.url.next** - Ein Link zur nächsten Seite der Sammlung
```